### PR TITLE
Fix element registration

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

This PR addresses one bug and updates doc/demo support to use markdown format. 

**Fixes:** #95

## Summary:

- docs: update pull request template
- docs: convert demo to markdown page|
- chore: update packages
- fix: #95 Missing conditional check

## Type of change:

Please delete options that are not relevant.

- [ ] New capability 
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Fix bug 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by littleninja — COMMENTED** [review]
I’m aware of changes with Rollup and demo docs but I’m not sure I’m familiar enough to approve.

File: demo/index_old.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```
Comment (geoffrich) [inline demo/index_old.html:1]: Can we delete this file?

**Review by geoffrich — COMMENTED** [review]
One comment, otherwise changes look good.

File: demo/index_old.html (hunk 1 above)
Comment (blackfalcon) [reply demo/index_old.html]: deleted

**Review by ecommerceUXdesign — APPROVED** [review]

**Pull request merged**